nohup.out

```diff
@@ -0,0 +1,8 @@
+events.js:167
```

Comment:
this file should not be checked in

Reply:
Will fix.  Should I add `nohup.out` to `.gitignore`?

Reply:
i don't know where this file came from, but either delete it or add to .gitignore

Reply:
It came from the `nohup` command.  `nohup.out` is a common generated file name, so it should go in `.gitignore`, which I did.